compatibility_server/pip_checker.py

```diff
@@ -260,7 +254,7 @@ def _run_container(self,
             self._mmap.record(self._tmap)
             raise PipCheckerError(
                 error_msg="An error occurred while starting a docker "
-                          "container. Error message: {}".format(e.explanation))
+                "container. Error message: {}".format(e.explanation))
```

Comment:
I think this would be weird - it makes more sense to use parens
```
                error_msg=("An error occurred while starting a docker "
                           "container. Error message: {}").format(e.explanation))
```
or break that line up by defining `error_msg` on a separate line beforehand.

Reply:
Actually, parens were required before: http://google.github.io/styleguide/pyguide.html#32-line-length

I can fix this everywhere but first we should decide whether we want to do this at all.